src/app/server/Server.h

```diff
@@ -369,6 +369,18 @@ class Server
 
 #if CHIP_CONFIG_ENABLE_ICD_SERVER
     app::ICDManager & GetICDManager() { return mICDManager; }
+
+#if CHIP_CONFIG_ENABLE_ICD_CIP
```

Comment:
I have no idea what CIP is.  Check-In Protocol?  If so, let's just do CHIP_CONFIG_ENABLE_ICD_CHECK_INS or something?

Reply:
The idea was to mimick the feature map naming. 
CIP -> Check-In Protocol
UAT -> User ActiveMode trigger
LITS -> LIT support.

If the preffered option is to spell the "feature", we should also do the other ones.

Reply:
I think it's very non-obvious that these defines are related to the feature codes (not names) for the spec....  Spelling them out would be a lot more readable, imo.

Reply:
In that case, to avoid convulating this PR with all the rename changes, I'll do it in a follow to simplify the reviews of this PR.

Reply:
Yep, that's fine.